Update Installation_iOS_6.md (#5218)
vcmi/vcmi: docs/players/Installation_iOS.md

### Installing the IPA File
- In AltStore: Go to **My Apps**, press the **+** button, and select `VCMI-iOS.ipa` to install.
- Using iTunes: Drag and drop the IPA file into your iOS device.

Alternatively, use Xcode or Apple Configurator (available for free on the Mac App Store). Apple Configurator also allows command-line installations. Here’s an example:

```sh
/Applications/Apple\ Configurator.app/Contents/MacOS/cfgutil install-app ~/Desktop/vcmi.ipa
```

## Alternative Step 2: Installing Heroes III Data Files

**Note:** To save space and time, you can skip downloading the `VIDEO.VID` file from the **Data** folder if you don’t need in-game videos. You can also skip the **Mp3** directory.

### Step 2.a: Using Finder or Windows Explorer

1. Upload the following directories to your device:
   - **Data**
   - **Maps**
   - **Mp3** (optional)

2. Use Finder (macOS) or iTunes (Windows/macOS 10.14 or earlier). Mods can also be added by uploading the **Mods** directory.

> 3. Follow [Apple’s guide](https://support.apple.com/en-us/HT210598) to place the files into the VCMI app. **Finder does not display copy progress**, so wait about 10 minutes for the process to finish.

Reviewer: ```suggestion
3. Follow [Apple’s guide](https://support.apple.com/en-us/HT210598) to place the files into the VCMI app. **Finder does not display copy progress in old macOS versions**, so wait about 10 minutes for the process to finish.
```

You can start game directly (avoiding the launcher) by changing setting in iOS Settings app - VCMI.

## Troubleshooting: Keeping your Alternative Store updated

Every 7 days you need to open Alt Store or the alternative. Clicking Update All or click Update App on both VCMI and the store separately works

If you iTunes does not connect to your device over WiFi despite enabling the Sync over WiFi option, do the following on PC:

> a) Search for ‘cmd’. Right click on it and ‘Run as administrator’.

Reviewer: ```suggestion
a) Search for `cmd`. Right click on it and ‘Run as administrator’.
```

You can start game directly (avoiding the launcher) by changing setting in iOS Settings app - VCMI.

## Troubleshooting: Keeping your Alternative Store updated

> If you iTunes does not connect to your device over WiFi despite enabling the Sync over WiFi option, every 7 days you need to open Alt Store or the alternative. Clicking Update All or click Update App on both VCMI and the store separately works. Do the following on PC:

Reviewer: ```suggestion
If iTunes does not connect to your device over WiFi despite enabling the Sync over WiFi option, every 7 days you need to open Alt Store or the alternative. Clicking Update All or click Update App on both VCMI and the store separately works. Do the following on PC:
```

1. Search for `cmd`. Right click on it and ‘Run as administrator’.
2. Copy the command below. It stops the ‘Apple Mobile Device service’.

>     net stop "Apple Mobile Device Service"

Reviewer: ```suggestion
       net stop "Apple Mobile Device Service"
```

> a) Search for `cmd`. Right click on it and ‘Run as administrator’.
> b) Copy the command below. It stops the ‘Apple Mobile Device service’.
> 
> net stop "Apple Mobile Device Service"
>   
> c) Copy the command below and restart the ‘Apple Mobile Device Service’.
> 
> net start "Apple Mobile Device Service"

Reviewer: ```suggestion
1. Search for `cmd`. Right click on it and ‘Run as administrator’.
2. Copy the command below. It stops the ‘Apple Mobile Device service’.

    net stop "Apple Mobile Device Service"
  
3. Copy the command below and restart the ‘Apple Mobile Device Service’.

    net start "Apple Mobile Device Service"
```

>     net start "Apple Mobile Device Service"

Reviewer: ```suggestion
       net start "Apple Mobile Device Service"
```

> In the event that you don't update the store in the alloted time and it expires, it won't load. Reloading the store is easy enough. Simply

Reviewer: ```suggestion
In case you don't update the store in the alloted time and it expires, it won't load. Reloading the store is easy enough. Simply
```

1. Search for `cmd`. Right click on it and ‘Run as administrator’.
2. Copy the command below. It stops the ‘Apple Mobile Device service’.

> net stop "Apple Mobile Device Service"

Reviewer: the whole point of those leading spaces was to make this command look like code block

Author: When I tried to push this through it failed though. happy to try again with your edit

Reviewer: > When I tried to push this through it failed though. happy to try again with your edit

ah, sorry, didn't see that it had failed
